### PR TITLE
[NETBEANS-5057] improved help on JSF composite component attributes

### PR DESCRIPTION
**Pull request by fcorneli**
This PR improves the help on JSF (composite) component attributes.
Also visualizes the type of the attribute, which is what I was always missing during development.

https://issues.apache.org/jira/browse/NETBEANS-5057

### CODE REVIEW

File: enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/facelets/CompositeComponentLibrary.java [hunk 1]
```diff
@@ -250,9 +250,11 @@ private synchronized void load() {
                 for (Map<String, String> attrsMap : model.getExistingInterfaceAttributes()) {
                     String attrname = attrsMap.get("name"); //NOI18N
                     boolean required = Boolean.parseBoolean(attrsMap.get("required")); //NOI18N
-                    String attributeDescription = getAttributesDescription(model, true);
+                    String attributeDescription = attrsMap.get("shortDescription"); //NOI18N
```
Comment (matthiasblaesing) [inline enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/facelets/CompositeComponentLibrary.java:253]: Could you please describe what this change means? A visualization would be helpful.

Comment (fcorneli) [reply enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/facelets/CompositeComponentLibrary.java:253]: With `getAttributesDescription` you used to get the description of all attributes on each individual attribute, without any distinction. Hence the full list all over the place. By using `shortDescription`, you really get just the description of the corresponding attribute, which is what you actually want to see.

File: enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/completion/JsfCompletionItem.java [hunk 2]
```diff
@@ -260,6 +260,11 @@ private String getHelpContent() {
             sb.append("<h1>"); //NOI18N
             sb.append(attr.getName());
             sb.append("</h1>"); //NOI18N
+            if (attr.getType() != null) {
+                sb.append("<div><b>Type:</b> "); //NOI18N
```
[the remark this answers is not in the document]
Comment (fcorneli) [reply enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/completion/JsfCompletionItem.java:264]: I'll have a look at the i18n handling.

**Comment by matthiasblaesing** [comment]
Merged via https://github.com/apache/netbeans/commit/fcdaae816f9cd7bfc8fed39f605165d553a6c165. Thank you!

**Pull request closed without merging**